content/en/blog/_posts/2025-12-17-kubernetes-v1-35-release/index.md

```diff
@@ -0,0 +1,450 @@
+
+There are also some [deprecations and removals](#deprecations-and-removals) in this release; make sure to read about those.
+
+## Release theme and logo
+
+{{< figure src="TBD" alt="Kubernetes v1.35 logo: TBD" class="release-logo" >}}
+
+TBD
+
+## Spotlight on key updates
+
+Kubernetes v1.35 is packed with new features and improvements. Here are a few select updates the Release Team would like to highlight!
+
+### Stable: In-place update of Pod resources
+
+Kubernetes is graduating in-place updates for Pod resources to General Availability (GA).
+This feature allows users to adjust cpu and memory resources without restarting Pods or Containers. Previously, such modifications required recreating Pods, which could disrupt workloads, particularly for stateful or batch applications. Previous Kubernetes releases already allowed you to change infrastructure resources settings (requests and limits) for existing Pods. This allows for smoother vertical scaling, improves efficiency, and can also simplify development.
+
+This work was done as part of [KEP #1287](https://kep.k8s.io/1287) led by SIG Node.
+
+### Beta: Pod certificates for workload identity and security
+
+Previously, delivering certificates to pods required external controllers (cert-manager, SPIFFE/SPIRE), CRD orchestration, and Secret management, with rotation handled by sidecars or init containers. KEP-4317 enables native workload identity with automated certificate rotation, drastically simplifying service mesh and zero-trust architectures. 
+Now, the `kubelet` generates keys, requests certificates via PodCertificateRequest, and writes credential bundles directly to the pod's filesystem. `kube-apiserver` enforces node restriction at admission time, eliminating the single biggest footgun for third-party signers: accidentally violating node isolation boundaries. Pure mTLS flows, no bearer tokens in the issuance path.
+
+This work was done as part of [KEP #4317](https://kep.k8s.io/4317) led by SIG Auth.
+
+### Alpha: Node declared features before scheduling
+
+When control planes enable new features, but nodes lag behind (permitted by Kubernetes skew policy), the scheduler can place pods requiring those features onto incompatible older nodes.
+The node-declaration features framework: nodes declare their supported Kubernetes features. With the new alpha feature enabled, a Node reports the features it supports, publishing this information to the control plane via a new `.status.declaredFeatures` field. Then, the `kube-scheduler`, admission controllers and third-party components can use these declarations. For example, you can enforce scheduling and API validation constraints to ensure that Pods run only on compatible nodes.
+
+This work was done as part of [KEP #5328](https://kep.k8s.io/5328) led by SIG Node.
+
+## Features graduating to Stable
+
+*This is a selection of some of the improvements that are now stable following the v1.35 release.*
+
+### Comparable resource version
```

Comment:
Hello @kubernetes/sig-api-machinery-leads @michaelasp! Our release comms team put this section together for https://kep.k8s.io/5504 for the v1.35 announcement blog which will be shared with the media for an early peak by tomorrow, and of course, which we're sharing with the media for an early peak, and of course, we release next week.

Appreciate any contribution, advice or feedback 😄